Felienne/hedy
# [PERFORMANCE] Caches quizzes on server start

**Description**
In this PR we improve the quiz implementation for dynamic keywords parsing. We follow the new structure implemented for the adventures and commands in #2493. This means that all language/keywords combinations are automatically created and cached on server start. However, when we are developing this caching is prevented to make sure the server starts (and restarts) quickly. This implementation has the advantage that after a longer server start-up the performance of the website will be much quicker and the server and memory load will be lower.

**The keyword language is currently fixed to English until the questions are fixed for dynamic keywords.**

**Fixes**
This PR fixes #2518.

**How to test**
This is just a performance task. Verify that the server is start much faster locally in comparison with the current main branch (as we still always cache on start-up there). Also verify that the quiz is working correctly for all languages.
